concepts/regular-expressions/about.md

```diff
@@ -0,0 +1,160 @@
+# About
+
+## 1. Regular Expressions in JavaScript
+
+A [Regular Expression][mdn-regular-expressions] (or Regex) is a sequence of characters that we can use to target and manipulate certain elements in strings. Hence, we can:
+
+- **Search** for a text in a string
+- **Replace** substrings in a string
+- **Extract** information from a string
+
+> 💡 JavaScript's regex flavor is part of the ECMA-262 standard for the language. This means that you don't have to worry about browser-specific compatibility.
+
+## 2. How to create Regular Expressions
+
+1. **Regular Expression Literal**:
+
+```javascript
+const regex = /[a-z]/;
+```
+
+2. Constructor **RegExp**:
+
+```javascript
+const regex = new RegExp('[a-z]');
+```
+
+## 3. 🏴‍☠️ Flags
+
+Some of the widely used are:
+
+- `/g` - Global Search
+- `/i` - Case Insensitive
+- `/m` - Multiline Search
+
+Here is a simple example:
+
+```javascript
+const re = /home/gi;
+const str = 'Home, sweet home.';
+const myArray = str.match(re);
+console.log(myArray);
+
+=> // ["Home", "home"]
```

Comment:
We often use this REPL style notation to make examples more concise:

```suggestion
str.match(re);
// => ["Home", "home"]
```

So just an expression and then `// =>` in the next line followed by the result of the expression.

It is also ok to keep your console log version but `=> //` needs to be changed to `// =>` in any case.